etcdserver/api/v3rpc/watch.go

```diff
@@ -65,7 +74,7 @@ func (ws *watchServer) Watch(stream pb.Watch_WatchServer) error {
 	}
 }
 
-func sendLoop(stream pb.Watch_WatchServer, watchStream storage.WatchStream, closec chan struct{}) {
+func sendLoop(stream pb.Watch_WatchServer, watchStream storage.WatchStream, ctrlStream <-chan *pb.WatchResponse, closec chan struct{}) {
```

Comment:
I feel like the arguments here should be a new `struct` of part of an existing `struct` (fields in watchServer? `go ws.sendLoop()` seems somewhat "natural" to me)